tools/qlog/src/lib.rs

```diff
@@ -1456,6 +1456,7 @@ pub enum EventData {
     H3DataMoved {
         stream_id: String,
         offset: Option<String>,
+        length: Option<u64>,
```

Comment:
What's the logic behind using `String` in some places but `u64` in others? IIRC you said `String` is for big numbers that can't be encoded in JSON, but using both makes the API pretty confusing.

Not really related to this PR, but in the future we should just make all these fields `u64` and then tag the ones that should be encoded as string with the `#[serde(serialize_with = "u64_to_str")] ` attribute, where `u64_to_str` is a function we implement to automatically encode specific fields as string. This way at least the API exposed by the `qlog` crate doesn't leak limitations in the underlying encoding format.

Reply:
The qlog crate's datamodel follows closely the real qlog datamodel, which has inconsistencies like this throughout the schema. 

I believe these will be fixed across the board in qlog-02, so we can fix stuff up then and consider custom serialize/deserialize functions.